config/config.go

```diff
@@ -83,12 +82,12 @@ func ReadConfigFile(filename string) (*Config, error) {
 
 // Validate checks the configuration for common errors.
 func (c Config) Validate() error {
-	for _, thread := range c.Threads {
-		if _, err := os.Stat(thread.PacketsDirectory); err != nil {
```

Comment:
as explained in 180fd9b, checking for the directories prevents stenographer from creating the directories itself (`thread/thread.go`, function `makeDirIfNecessary()`).